clustered/client/src/main/java/org/ehcache/clustered/config/builder/ClusteringServiceConfigurationBuilder.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright Terracotta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.ehcache.clustered.config.builder;
```

Comment:
The package in impl is `org.ehcache.impl.config.builders` - should we align here on the plural as well? Knowing that a cache level config might be coming.